alacritty/src/window_context.rs

```diff
@@ -183,9 +188,12 @@ impl WindowContext {
             self.display.pending_update.dirty = true;
         }
 
-        // Live title reload.
-        if !config.window.dynamic_title || old_config.window.title != config.window.title {
-            self.display.window.set_title(&config.window.title);
+        // Live title reload only when title wasn't provided on command line.
+        //
+        // We know that the title was provided on the command line when dynamic_title is disaabled
+        // and old_config title isn't equal to the current one.
+        if !config.window.dynamic_title && self.identity.title != old_config.window.identity.title {
```

Comment:
The `||` was in originally to allow changing the title when it was still the same as the original window title. Is this not something we want? I feel like it makes sense but ultimately I don't really care.

Reply:
I mean it should be changed if it was fully provided by the config. Since we compare with old title which should match the old config title in case title change isn't allowed. The case is that it doesn't only work when you pass title on CLI in other cases it'll work.